STYLEGUIDE.md

```diff
@@ -29,13 +47,37 @@ import { intro, outro } from '../../utils/styles/index.js'
 export const basicCommand = () => {
   intro('basic command')
   // do stuff
-  outro()
+  outro({ message: 'Your message', exit: true })
```

Comment:
If `exit: true` were not passed, what would happen?

Reply:
It would just print the outro message. I think the signature of this was updated recently so this just updates it to the current signature. I am intrigued as to whether exit should always just happen otherwise clack will just look weird, but will probably tackle that separately.

Reply:
Agree with your other comment on the outro bit. Will circle back to that as I agree